src/test/java/games/strategy/net/MessengerLoginTest.java

```diff
@@ -1,146 +0,0 @@
-package games.strategy.net;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
-import static org.mockito.Mockito.mock;
-
-import java.net.SocketAddress;
-import java.util.HashMap;
-import java.util.Map;
-
-import org.junit.Before;
-import org.junit.Test;
-
-import games.strategy.engine.ClientContext;
-import games.strategy.engine.framework.startup.login.ClientLogin;
-import games.strategy.engine.framework.startup.login.ClientLoginValidator;
-import games.strategy.test.TestUtil;
-import games.strategy.util.MD5Crypt;
-
-public class MessengerLoginTest {
```

Comment:
The tests in this class were replaced with new unit and integration tests:

* The `testSimple` and `testRefused` scenarios are covered by the new unit tests in `ClientLoginTests` and `ClientLoginValidatorTests`.
* The `testGetMagic` scenario is useless and was removed.
* The `testPassword` scenario is covered by the new integration tests in `ClientLoginIntegrationTest`.